docs/content/docs/faq.md

```diff
@@ -2,6 +2,20 @@
 title: FAQ
 ---
 
+## ``[BABEL] Note: The code generator has deoptimised the styling of ...``
+
+_This is not an issue. Just a simple information._
+
+When babel compiles some code, it try to offer a readable output, but when files
+become big (>100KB), babel considers (by default) that it's not useful to keep
+a readable output.
+If you are annoyed by this message, or if you don't care about readability,
+you can just use `compact: true`.
```

Comment:
Maybe. But I don't think "true" or "false" are better. Auto is a good choice. But why not forcing true by default.